### PR TITLE
RTL Chosen Select causes the page to disappear

### PR DESCRIPTION
**Pull request by demis-palma**
This fixes once again the problem explained in https://github.com/joomla/joomla-cms/pull/7106 which has returned after https://github.com/joomla/joomla-cms/pull/11779

@dgt41 was right. If we plan to further update the chosen codebase, we should move the hacks (if possible) into a different file.

### CODE REVIEW

**Comment by brianteeman** [comment]
Is this the same issue as #13861 reported by @PhilETaylor - sorry I cant test

**Comment by demis-palma** [comment]
To me it's a different problem. I have reproduced it on Widows with Google Chrome, Firefox and Internet Explorer. Opera browser is not affected.

**Comment by demis-palma** [comment]
@brianteeman you confirmed this the last time https://github.com/joomla/joomla-cms/pull/7106#issuecomment-109459018

**Comment by Saeedghb** [comment]
Thank you @demis-palma 

I had the same problem with the front-end select lists in rtl mode.
The issue can be verified by choosing beez3 as default template in front-end.

This commit fixed my problem.

**Comment by infograf768** [comment]
has this been solved in staging?

**Comment by sinahaghparast** [comment]
Hello,

It's work...


Apply the chzn-rtl class on your select
[github.com/harvesthq/chosen/issues/1056#issuecomment-13942263](https://github.com/harvesthq/chosen/issues/1056#issuecomment-13942263)

Chosen already supports right to left select boxes. just add chosen-rtl to your select tags:
<select class="chosen-select chosen-rtl">
[github.com/harvesthq/chosen/issues/1737#issuecomment-32887958](https://github.com/harvesthq/chosen/issues/1737#issuecomment-32887958)

==========

**Comment by Saeedghb** [comment]
@sinahaghparast 
As I tested, neither class="chzn-rtl" nor class="chosen-select chosen-rtl" does not solve the problem with Joomla! 3.8.12 and beez3 template.

Did you test it with beez3 template?

**Comment by sinahaghparast** [comment]
when you load list form... Joomla load media/jui/css/chosen.css automaticly...
which template is not important...

in this file at line 368 you can see, 
.chzn-rtl {
  text-align: right;
}
=====
We are together...

**Comment by Saeedghb** [comment]
@sinahaghparast

Of course I can see the above mentioned code, but adding "chzn-rtl" class to select list does not solve the problem.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/16552">issues.joomla.org/tracker/joomla-cms/16552</a>.</sub>

**Comment by Saeedghb** [comment]
The problem is present with K2 filter module.
This commit does not solve the problem.

**Comment by brianteeman** [comment]
best to ask the makers of k2 for problems with k2

**Comment by Saeedghb** [comment]
@brianteeman

This problem is not unique to K2. It is caused by a version of chozen used in Joomla!

**Comment by Saeedghb** [comment]
> best to ask the makers of k2 for problems with k2

@brianteeman

This problem is not unique to K2. It is caused by a version of chozen used in Joomla!

**Comment by laoneo** [comment]
Does the issue still exists with the current Joomla 3 code base only (without K2)? If not then it got solved along the line since this pr was created.

**Comment by brianteeman** [comment]
Yes it still exists. Easy to test using the instructions from the linked issues

arabic

eng

**Comment by brianteeman** [comment]
I have tested this item :white_check_mark: successfully on ed2cd133ce4afa3bf62cbaac295fb8f1e038030f<br /><br />forgot to add that the pr solves the issue<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/16552">issues.joomla.org/tracker/joomla-cms/16552</a>.</sub>

**Comment by laoneo** [comment]
I have tested this item :white_check_mark: successfully on 79c08b7ba03eb99932961676668c204ef050ed0e<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/16552">issues.joomla.org/tracker/joomla-cms/16552</a>.</sub>

**Comment by richard67** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/16552">issues.joomla.org/tracker/joomla-cms/16552</a>.</sub>

**Comment by zero-24** [comment]
I'm aware that I'm late to this party beeing that old but why do we comment a CSS rule and not just remove it?

**Comment by laoneo** [comment]
I would also remove the comments.

**Comment by richard67** [comment]
@zero-24 @laoneo So shall we remove RTC and add the"Updates Requested" label? I worry the author could not be available anymore for making changes.

File: media/jui/css/chosen.css [hunk 1]
```diff
@@ -402,7 +402,8 @@ This file is generated by `grunt build`, do not edit it by hand.
   right: auto;
   left: 4px;
 }
-.chzn-rtl.chzn-container-single-nosearch .chzn-search,
+/* https://github.com/joomla/joomla-cms/pull/7106
+.chzn-rtl.chzn-container-single-nosearch .chzn-search, */
```
Comment (laoneo) [inline media/jui/css/chosen.css]:
```suggestion
```

File: media/jui/css/chosen.css [hunk 2]
```diff
@@ -429,5 +430,13 @@ This file is generated by `grunt build`, do not edit it by hand.
 .chzn-rtl.chzn-container-single.chzn-with-drop .chzn-single div b {
   background-position: -12px 2px;
 }
-
+/* https://github.com/joomla/joomla-cms/pull/7106 */
```
Comment (zero-24) [inline media/jui/css/chosen.css]:
```suggestion
/* Taken from https://github.com/joomla/joomla-cms/pull/16552 */
```

Comment (laoneo) [inline media/jui/css/chosen.css]:
```suggestion
```

**Comment by zero-24** [comment]
Merging here thanks for the patience

**Pull request merged**